proto/structured.proto

```diff
@@ -104,3 +105,220 @@ message CreateTableResponse {
 // service SchemaService {
 //   rpc CreateTable (CreateTableRequest) returns (CreateTableResponse);
 // }
+
+// TableRequestHeader is supplied with every structured API request.
+message TableRequestHeader {
```

Comment:
I had originally done it that way, but with all the requests/responses having Table in their name it looked awkward